torch/onnx/symbolic_opset10.py

```diff
@@ -206,38 +206,12 @@ def embedding_bag(g,
     if scale_grad_by_freq and sym_help._training_mode:
         return sym_help._onnx_unsupported('embedding_bag with scale_grad_by_freq for training mode')
 
-    from torch.onnx.symbolic_opset9 import size, div, select
```

Comment:
Why are these removed? Was it that the graph changes certain assumptions don't stand anymore?

Reply:
No, actually the previous export logic was buggy for some cases.
The behavior of the embedding_bag functional module is different when input is 2D or 1D. The code here detected this case by checking for this condition in symbolic through accessing the input to indices node, like below:
```if len(indices.node().inputs().__next__().type().sizes()) == 2```
But then I saw other use-cases of embedding_bags, when this condition was a false position.
I'm removing this part of the code (which was mainly for dynamic export of this op with inputs of different shape) since its logic is broken, and I'm adding full dynamic support as part of opset 11.